### PR TITLE
two-fer: Update description.md

### PR DESCRIPTION
**Pull request by yyyc514**
I don't think it's great anyways but I'm tired of seeing students submit X as the variable name because of picking up some subliminal hint here.

### CODE REVIEW

**Comment by yyyc514** [comment]
No one returned a literal X... they'd do `twoFer(X="you")`

**Comment by coriolinus** [comment]
Heh. Definitely doesn't hurt to suggest a more sensible name, then.

**Comment by cmccandless** [comment]
See #1470

**Comment by yyyc514** [comment]
If we replace it we do, but no reason not to improve it until then. :-)

File: exercises/two-fer/description.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
 Given a name, return a string with the message:
 
 ```text
-One for X, one for me.
+One for [name], one for me.
 ```
 
-Where X is the given name.
+Where "name" is the given name.
```
Comment (yawpitch) [inline exercises/two-fer/description.md]: Almost guarantee you you're going to see submissions that return 'One for "name", one for me.'.

Comment (yyyc514) [reply exercises/two-fer/description.md]: We'll see.  I'm not worried.

Comment (yyyc514) [reply exercises/two-fer/description.md]: More people use X than do some other weird type of misunderstanding...

File: exercises/two-fer/description.md [hunk 2]
```diff
@@ -3,10 +3,10 @@
 Given a name, return a string with the message:
 
 ```text
-One for X, one for me.
+One for [name], one for me.
```
Comment (kotp) [inline exercises/two-fer/description.md]:
```suggestion
One for name, one for me.
```

**Comment by yyyc514** [comment]
I'm fine with whatever anyone decides that removes the X.

**Comment by kotp** [comment]
If I could, I would merge it in, but no rights to do so.

**Comment by kotp** [comment]
Or it is being blocked because of the Travis-CI saying "Expected -- Waiting fo rstatus to..." but it has been that way for hours.

**Comment by yyyc514** [comment]
Sometimes Travis-CI just dies or the API or connection or something.  Really someone needs to build in some failsafe for these things that block.

**Comment by kotp** [comment]
Maybe make squash the commits, and force push the branch to kickstart Travis again?

**Comment by yawpitch** [comment]
You can also make an empty commit and push that, if you’re not comfortable with squashing, ie `git commit --allow-empty -m “Bump CI”`.

**Comment by kotp** [comment]
> You can also make an empty commit and push that, if you’re not
> comfortable with squashing, ie
> `git commit --allow-empty -m “Bump CI”`.

Great point.  Do you know if force pushing the same commit will also
trigger a new run of the CI?

**Comment by yawpitch** [comment]
> You can also make an empty commit and push that, if you’re not comfortable with squashing, ie `git commit --allow-empty -m “Bump CI”`.
> Great point. Do you know if force pushing the same commit will also trigger a new run of the CI?

It should do, yeah.

**Comment by yyyc514** [comment]
>  Do you know if force pushing the same commit will also trigger a new run of the CI?

Force pushing the same commit is really a NOOP if the SHA1 hasn't changed, is it not?

**Comment by rpottsoh** [comment]
I re-ran the build.  Appears to be ready to merge now.

**Pull request merged**